domain/src/main/java/com/glima/domain/business/usecase/UseCase.kt

```diff
@@ -0,0 +1,7 @@
+package com.glima.domain.business.usecase
+
+import kotlinx.coroutines.flow.Flow
+
+interface UseCase<in P, R> {
```

Comment:
After finishing coding this app maybe I have some regrets using flow 😅 . I like to have a contract across all use cases, I think ```suspend fun execute(params: P): R ``` probably resolves well my problems.